redhat-cop/infra-ansible
# Fixing dns zone role to avoid clobbering existing tsig keys

### What does this PR do?
The `dns/manage-dns-zones-bind` role didn't check correctly if a tsig key existed for a specific view/zone and hence always re-generated the key even one when existed. This PR fixes this issue + a few other minor issues:

- Fix check for existing tsig key
- Fix Ansible >=2.8 use of `import_tasks` in handlers to use `include_tasks`

### How should this be tested?
1. Run through a complete install/configuration of a DNS server with one or more views/zones.
2. Add another zone to the inventory, re-run the configuration. Verify that existing keys didn't change, but the new one was added.

### Is there a relevant Issue open for this?
N/A

### Other Relevant info, PRs, etc.
N/A

### People to notify
cc: @redhat-cop/infra-ansible

@ostefek99 @paulbarfuss FYI
